pkg/frontend/transport/handler_test.go

```diff
@@ -199,6 +220,13 @@ func TestHandler_ServeHTTP(t *testing.T) {
 				require.EqualValues(t, 0, msg["split_queries"])
 				require.EqualValues(t, 0, msg["estimated_series_count"])
 				require.EqualValues(t, 0, msg["queue_time_seconds"])
+
+				if tt.expectedReadConsistency != "" {
+					require.Equal(t, tt.expectedReadConsistency, msg["read_consistency"])
+				} else {
+					_, ok := msg["read_consistency"]
+					require.False(t, ok)
+				}
```

Comment:
nit: this can become just one line and it should still pass, no? `require.Equal(t, tt.expectedReadConsistency, msg["read_consistency"])`

Reply:
I wanted to test that we don't include `read_consistency=""` when it's not explicitly set. That's what this assertion does.